site/en/posts/pwa-summit-2022.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Building the future of web apps: Highlights from PWA Summit 2022'
+metadesc: With talks on PWA foundation, nuts and bolts for practitioners, inspiration from global brands, and improvements to the ecosystem, recap everything that happened at PWA Summit 2022!
+tags:
+  - event
+  - web
+hero:
+  youtube: m-2lLWI_csE
+  alt: PWA Summit 2022, block 1
+authors:
+  - samrichard
+date: 2022-11-04
+---
+
+What sets the web apart from every other development platform? It belongs to all of us. The web’s communal nature is what helps us realize its potential, and no one knows this better than web devs. They enhance the web bit by bit every day to help all of us build better experiences for [more than 5 billion internet users worldwide.](https://www.statista.com/statistics/617136/digital-population-worldwide/) And Progressive Web Apps (PWAs) that make the most of the web’s capabilities while feeling “appy” are a big part of this effort.
```

Comment:
nit: Isn't starting a sentence with "And" a grammatical no-no?

Reply:
I'd generally agree here, but this copy is shared between the companies that put the event on and is locked so I don't want to go rewording stuff like this, if that's OK.

Reply:
Yeah, no worries then.